README.rst

```diff
@@ -131,6 +131,12 @@ Load form (with ``SMUGGLER_FIXTURE_DIR`` configured):
 Release notes
 =============
 
+Version 0.9.0 (2018-??-??)
```

Comment:
Hey @jaap3, I was thinking in releasing version 0.8 as 1.0 (bug fixes if we ever need on 1.0 branch, then minor bugfixes releases on 1.0.x).

The current master we release as 1.1.

What do you think?

Reply:
Actually, better we release it as 0.9. We let it mature than tag it as 1.0.

If we ever add new features at futre, it should go to 1.1 and the current 0.9.x tag turns in 1.0.